### PR TITLE
doc/admin/updates: add file for docker-compose manual migrations

### PR DESCRIPTION
**Pull request by slimsag**
This file will document any manual steps needed for docker-compose migrations.

In the near future, I also plan to move https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/migrate.md into a file on our docsite here as well.

### CODE REVIEW

File: doc/admin/updates/docker_compose.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+### Manual migration step: adjust file permissions
+
+Please adjust the redis-store and redis-cache volume permissions by running the following on the host machine:
+
+```
+docker run --rm -it -v /var/lib/docker:/docker alpine:latest sh -c 'chown -R 999:1000 /docker/volumes/docker-compose_redis-store /docker/volumes/docker-compose_redis-cache'
+```
+
+### Upgrade
```
Comment (ggilmore) [inline doc/admin/updates/docker_compose.md]: We strongly recommend that users [fork this repository](https://docs.sourcegraph.com/admin/install/docker-compose#optional-recommended-store-customizations-in-a-fork) and merge tagged releases into their `release` branch (akin to what we recommend for deploy-sourcegraph). That should be accounted for here.

Comment (slimsag) [reply doc/admin/updates/docker_compose.md]: I just copied this from what we already had documented in https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/86/files#diff-5daf043d0809feb2e518eeafd02d6ff1L26

Comment (slimsag) [reply doc/admin/updates/docker_compose.md]: Updated to reflect branching for now.

Comment (ggilmore) [reply doc/admin/updates/docker_compose.md]: You beat me to it. The only thing I'd ask is to add a note linking back to https://docs.sourcegraph.com/admin/install/docker-compose#optional-recommended-store-customizations-in-a-fork

**Pull request merged**